run_dir/static/js/worksets.js

```diff
@@ -42,7 +42,12 @@ function init_listjs() {
     var table = $('#workset_table').DataTable({
       "paging":false,
       "info":false,
-      "order": [[ 0, "desc" ]]
+      "order": [[ 0, "desc" ]],
+      dom: 'Bfrtip',
```

Comment:
Do we need all of these options? We don't use any pagination (p) for example

Reply:
I guess not, maybe it was used before? It was added [8 years ago](https://github.com/SciLifeLab/genomics-status/commit/1bed2aec756b8786fe824d112cd5ef9b179ad943).

Maybe we don't need `"info":false,` either?

Reply:
Ah! :-D

Reply:
I removed it, and it seems to work as before 👍🏻 

I can't say I fully understand the other table control elements, so I'm not sure which are needed. 
Obviously "B" and "t" are needed, I guess `r - processing display element` is needed? Maybe not `i - Table information summary`? 🤔